src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/DebugProfileEnumValuesGeneratorTests.cs

```diff
@@ -32,8 +32,13 @@ public class DebugProfileEnumValuesGenerator_Tests
 
             Mock<ILaunchSettingsProvider> moqProfileProvider = new Mock<ILaunchSettingsProvider>();
             moqProfileProvider.Setup(p => p.CurrentSnapshot).Returns(testProfiles.Object);
+            var moqThreadingService = new Mock<IProjectThreadingService>();
```

Comment:
I ended up using [IProjectThreadingServiceMock](https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/IProjectThreadingServiceMock.cs) but I don't think the tooling concerns matter here.  If you are worried about trying to find implementations in the codebase, there are none outside of Mocks.  `IProjectThreadingService` is implemented by the Editor.  The pattern for the project system is to use mocks for all the external interfaces that are not implemented by the project system.  I agree we should use this pattern for interfaces that we export and implement.